packages/reports/addon/consumers/request/column.ts

```diff
@@ -59,6 +59,7 @@ export default class ColumnConsumer extends ActionConsumer {
       const { routeName } = route;
       const { request } = route.modelFor(routeName) as ReportModel;
       request.removeColumn(column);
+      this.requestActionDispatcher.dispatch(RequestActions.REMOVE_ROLLUP_COLUMN, route, column);
```

Comment:
Can we move this to a handler in `consumers/request/rollup.ts` that listens to `[RequestActions.REMOVE_COLUMN_FRAGMENT]` as done in the past:

https://github.com/yavin-dev/framework/blob/23a6258192842bd01d5507574267c9a2b5170c77/packages/reports/addon/consumers/request/sort.ts#L92